app/views/LocationShare/LiveLocationPreviewModal.tsx

```
@@ -0,0 +1,632 @@
+							<TouchableOpacity
+								disabled={submitting || !locationState?.coords}
+								onPress={onShare}
+								style={[styles.btn, styles.btnPrimary, !locationState?.coords && styles.btnDisabled]}>
+								{submitting ? (
+									<ActivityIndicator color='#fff' />
+								) : (
+									<Text style={[styles.btnText, styles.btnTextPrimary]}>Start</Text>
+								)}
+							</TouchableOpacity>
+						</>
+					)}
```

ESLint

[error] Do not nest ternary expressions.

(no-nested-ternary)